edx_figures/views.py

```diff
@@ -99,3 +105,23 @@ def get_queryset(self):
         queryset = super(UserIndexView, self).get_queryset()
 
         return queryset
+
+##
+## Views for edX Figures models
+##
+
+#class SiteDailyMetricsViewSet(CommonAuthMixin, viewsets.ModelViewSet):
```

Comment:
Remarked out when copied over from my sandbox dev. I will add Auth toward the end of this sprint after I get more of the metrics code in here